File: svc_Auth/Bank.Auth.App/Dto/UserDto.cs
using Bank.Auth.Common.Enumerations;

namespace Bank.Auth.App.Dto
{
    public class UserDto
    {
        public Guid Id { get; set; }
        public string Email { get; set; }
        public string? Name { get; set; }
        public List<Role> Roles { get; set; }
        public string Phone { get; set; }
        public bool IsBanned { get; set; } = false;
    }
}

File: client_Customer/app/src/main/java/com/example/customerclient/data/api/dto/UserDto.kt
package com.example.customerclient.data.api.dto

import com.example.customerclient.domain.models.UserInfo

data class UserDto(
    val id: String,
    val email: String?,
    val name: String?,
    val role: Int,
    val isBanned: Boolean
)

fun UserDto.toUserInfo(): UserInfo = UserInfo(id = this.id, name = this.name ?: "")
